test/Benchmarks/src/Warnings.enso

```diff
@@ -0,0 +1,81 @@
+from Standard.Base import all
+
+import Standard.Base.Errors.Illegal_Argument.Illegal_Argument
+from Standard.Base.Data.Array_Proxy import Array_Proxy
+
+from Standard.Test import Bench
+
+import project.Vector.Utils
+
+polyglot java import java.util.ArrayList
+
+type My_Error
+    Error msg
+
+options = Bench.options . set_warmup (Bench.phase_conf 1 3) . set_measure (Bench.phase_conf 1 3)
+
+type Data
+    Value ~no_warnings ~one_warning ~ten_percent_warnings ~all_warnings
+
+fmap f data = Data.Value (f data.no_warnings) (f data.one_warning) (f data.ten_percent_warnings) (f data.all_warnings)
```

Comment:
Nicer to make it an instance method `map` of `Data`. Then one would just use `data.map f`.